Build/Tactical/ShopKeeper_Interface.cc

```diff
@@ -2578,8 +2578,7 @@ static UINT32 CalcShopKeeperItemPrice(BOOLEAN fDealerSelling, BOOLEAN fUnitPrice
 		// if it's a GUN or AMMO (but not Launchers, and all attachments and payload is included)
 		switch ( GCM->getItem(usItemID)->getItemClass() )
 		{
-			case IC_GUN:
-			case IC_AMMO:
+			case IC_WEAPON:
```

Comment:
Is the removal of IC_AMMO on purpose? IC_WEAPON doesn't include it. Considering what it includes, the change seems excessive:
`define IC_WEAPON         ( IC_GUN | IC_BLADE | IC_THROWING_KNIFE | IC_LAUNCHER | IC_TENTACLES )`
So I'd prefer to just add IC_LAUNCHER to the cases.